model/storage/ConsumerStorage.php

```diff
@@ -73,21 +77,52 @@ public function setConsumerToken(\core_kernel_classes_Resource $consumer, Access
      *
      * Fetch token from cache if exists, otherwise set it
      *
-     * @param $tokenHash
+     * @param string $tokenHash
```

Comment:
I wont add typehint and return type in the scope of this PR, cause we have mixed behavior relying in boolean and other operators in different places.